integration-tests/src/test/resources/ldap_integration/readonlyrest.yml

```diff
@@ -123,6 +123,8 @@ readonlyrest:
       search_groups_base_DN: "ou=Groups,dc=example,dc=com"
       user_id_attribute: "uid"                                  # default "uid"
       unique_member_attribute: "uniqueMember"                   # default "uniqueMember"
+      group_id_attribute: "cn"                                  # default "cn"  
```

Comment:
Can we use the new format in the integration tests? IMO, the legacy format should be used only in the unit tests